apps/meteor/app/apps/server/converters/rooms.js

```diff
@@ -170,7 +170,11 @@ export class AppRoomsConverter {
 		};
 
 		if (!isPartial) {
-			Object.assign(newRoom, room._unmappedProperties_);
+			Object.assign(
+				newRoom,
+				{ t: room.type, ts: room.createdAt, msgs: room.messageCount || 0, _updatedAt: room.updatedAt },
```

Comment:
all of these are covered already with the new `typeof`s you add, I guess this is not needed?
```suggestion
```

Reply:
I put them here to ensure they would be always there 🤔 i can remove tho